src/routes/web.php

```diff
@@ -17,6 +17,7 @@
     Route::get('/post/{post}', [PostController::class, 'show'])->name('post.show');
     Route::get('/post/{post}/edit', [PostController::class, 'edit'])->name('post.edit');
     Route::put('/post/{post}', [PostController::class, 'update'])->name('post.update');
+    Route::delete('/post/{post}', [PostController::class, 'destroy'])->name('post.destroy');
```

Comment:
can't we maybe put in a resource all of the post?

Reply:
+ 1 for this one